examples/compose/compose.yml

```diff
@@ -11,7 +11,6 @@ services:
     volumes:
       - "./db:/var/lib/mysql"
     environment:
-      - "MYSQL_ALLOW_EMPTY_PASSWORD=yes"
```

Comment:
This is just an example and not necessary. Please see my comment in related issue https://github.com/anonaddy/docker/issues/272#issuecomment-2094487629

Reply:
"It is not necessary to make this secure on default." - Why, what's the hurt in this?

Reply:
As stipulated in https://github.com/anonaddy/docker/issues/272#issuecomment-2113493522 I don't mind if we set `MARIADB_RANDOM_ROOT_PASSWORD=yes` instead.

Also one of these is required anyway, you can just remove this env: https://hub.docker.com/_/mariadb

![image](https://github.com/anonaddy/docker/assets/1951866/73eeaaba-fde1-4cd9-a329-594df0519c21)